### PR TITLE
Fixed margin logical properties

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Description

convert margin into logical property

## Jira Ticket

- [THEMES-1731](https://arcpublishing.atlassian.net/browse/THEMES-1731)

## Test Steps

1. Checkout this branch `git checkout THEMES-1731-margin-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/medium-promo-block`

## Dependencies or Side Effects

- Feature pack pr for local development: https://github.com/WPMedia/arc-themes-feature-pack/pull/507

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-1731]: https://arcpublishing.atlassian.net/browse/THEMES-1731?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .storybook/themes/news.scss [hunk 1]
```diff
@@ -3903,7 +3903,10 @@
 						),
 						"media-item": (
 							"float": left,
-							"margin": 0,
+							"margin-block-end": 0,
+							"margin-block-start": 0,
+							"margin-inline-end": 0,
+							"margin-inline-start": 0,
 							"max-width": 33%,
```
Comment (vgalatro) [inline .storybook/themes/news.scss]: This should be `max-inline-size` like it is in `news.json`.

File: .storybook/themes/commerce.scss [hunk 2]
```diff
@@ -632,12 +632,12 @@
 					)
 				),
 				"hero-text-overlay": (
-					"margin": 0auto,
+					"margin": 0 auto,
```
Comment (vgalatro) [inline .storybook/themes/commerce.scss]: These values are incorrect in the `commerce.json` files of the blocks as well. I wouldn't worry about them for this ticket, but I do think we'll need one to convert the commerce only blocks to use logical properties.

**Review by vgalatro — CHANGES_REQUESTED** [review]
Just a small change needed, to keep the storybook SCSS in line with the JSON files.

File: .storybook/themes/commerce.scss (hunk 2 above)
Comment (malavikakoppula) [reply .storybook/themes/commerce.scss]: Thanks Vito, reverted all commerce changes

File: .storybook/themes/news.scss (hunk 1 above)
Comment (malavikakoppula) [reply .storybook/themes/news.scss]: Thanks Vito, updated this

**Review by vgalatro — APPROVED** [review]
LGTM!

**Pull request merged**